esp/platform/espcfg.ipp

```diff
@@ -202,14 +234,28 @@ public:
     void loadAll()
     {
         DBGLOG("loadServices");
-        loadServices();
+        try
+        {
+            loadServices();
+        }
+        catch(IException* ie)
+        {
+            if (isDetachedFromDali())
```

Comment:
Should you move this check to the top and not even try?  Otherwise maybe change the wording a little bit.  The exception could be for something else, in which case being detached from DALI is ALSO an error.  (Picky, but as simple as changing wording from "Could not" to "Can not".)

Reply:
@rpastrana ?

Reply:
The problem is that some ESP services can load while Detached from DALI, this message conveys the fact that in this particular system, we were not able to load ESP service(s) while in detached state. We don't know if the failure is due to the detached state either.
Perhaps rewording as such could be more accurate and helpful:
"Could not load ESP service(s) while DETACHED from DALI - Consider re-attaching ESP process"